dags/monrecap.py

```diff
@@ -23,11 +24,12 @@ def monrecap_airtable(**kwargs):
         con = db.connection_engine()
         # Need to drop these tables and the views created with them in order to be able to run the df.to_sql()
         con.execute(
-            """drop table if exists monrecap."Commandes" cascade;
+            """drop table if exists monrecap."Commandes_v0" cascade;
                     drop table if exists monrecap.barometre cascade;"""
         )
 
-        tables = ["Commandes", "Contacts", "Baromètre - Airflow"]
+        table_mapping = {"Commandes": "Commandes_v0", "Contacts": "Contacts_v0"}
+        tables = ["Commandes", "Contacts"]
         for table_name in tables:
```

Comment:
tu pourrais enlever ligne 32 et faire `for table_name in tables.keys()`

(bonne pratique pr pas avoir besoin de changer deux lignes si tu traites plus de tables demain)